src/pages/docs/messages/annotations.mdx

```diff
@@ -0,0 +1,529 @@
+
+
+## Annotation types <a id="annotation-types"/>
+
+Annotation types determine how annotations are processed and aggregated into [summaries](#annotation-summaries).
+
+The annotation type is a string of the format `namespace:summarization.version` where:
+
+- `namespace` is a string (e.g. `reactions`) that groups related annotations. Only annotations in the same namespace will be aggregated together to produce [summaries](#annotation-summaries).
+- `summarization` specifies the summarization method (e.g. `flag`) which determines how annotations are aggregated to produce [summaries](#annotation-summaries).
+- `version` specifies the version component of the summarization method (e.g. `v1`), which allows for future changes to summarization behavior.
+
+## Annotation summaries <a id="annotation-summaries"/>
+
+When annotations for a message are published, Ably automatically generates a summary that provides an aggregated view of all annotations for that message.
+
+A separate summary is produced for each distinct [annotation type](#annotation-types). The summarization method specified in the [annotation type](#annotation-types) determines how annotations in the same namespace for a given message are aggregated into a summary. A summary is constructed from the set of `annotation.create` and `annotation.delete` messages that have been published for a message.
+
+You can [subscribe](#subscribe-annotation-summaries) to these summaries using the `subscribe()` method on a channel, and they will be delivered as messages with an `action` of `message.summary`. The summary will be included on the message's `summary` field, which is an object whose keys are the [annotation types](#annotation-types) and whose values describe the annotation summary for that type:
+
+<Code>
+```json
+{
+  "metrics:total.v1": {
+    "total": 42
+  },
+  "reactions:flag.v1": {
+    "total": 3,
+    "clientIds": ["client1", "client2", "client3"]
+  },
+  "categories:distinct.v1": {
+    "important": {
+      "total": 2,
+      "clientIds": ["client1", "client3"]
+    },
+    "urgent": {
+      "total": 3,
+      "clientIds": ["client1", "client2", "client3"]
+    }
+  },
+  "status:unique.v1": {
+    "important": {
+      "total": 2,
+      "clientIds": ["client1", "client3"]
+    },
+    "urgent": {
+      "total": 1,
+      "clientIds": ["client2"]
+    }
+  },
+  "voting:multiple.v1": {
+    "option-a": {
+      "total": 7,
+      "clientCounts": {
+        "client1": 3,
+        "client2": 2
+      },
+      "totalUnidentified": 2
+    },
+    "option-b": {
+      "total": 4,
+      "clientCounts": {
+        "client1": 2,
+        "client3": 1
+      },
+      "totalUnidentified": 1
+    }
+  }
+}
+```
+</Code>
+
+
+When [publishing](#publish) an annotation you can optionally specify a `name` which is used by some summarization methods to produce a summary.
+
+Ably provides five summarization methods to support different use cases which are described below.
+
+### Total
```

Comment:
I wonder if these should be listed before the payload example, as they're key to understanding what type of annotation you can publish.